WeakAurasOptions/InformationOptions.lua

```diff
@@ -100,6 +100,34 @@ function OptionsPrivate.GetInformationOptions(data)
     }
     order = order + 1
   end
+  if OptionsPrivate.HasWagoUrl(data.id) then
+    args.ignoreWagoUpdate = {
+      type = "toggle",
+      name = L["Ignore Wago updates"],
+      desc = OptionsPrivate.IsWagoUpdateIgnored(data.id) and L["Do you want to stop ignoring all future updates for this aura"] or L["Do you want to ignore all future updates for this aura"],
+      width = WeakAuras.doubleWidth,
+      get = function() return OptionsPrivate.IsWagoUpdateIgnored(data.id) end,
+      set = function(info, v)
+          local auraData = WeakAuras.GetData(data.id)
+          if auraData then
+            for child in OptionsPrivate.Private.TraverseAll(auraData) do
+              if OptionsPrivate.IsWagoUpdateIgnored(data.id) then
+                child.ignoreWagoUpdate = nil
```

Comment:
This toggles individually on each aura, which is the wrong approach. You should toggle based on what IsWagoUpdateIgnored returns for the whole group.